src/sqlite/write.rs

```diff
@@ -156,9 +158,9 @@ impl DataSink for SqliteDataSink {
                     sqlite.delete_all_table_data(&transaction)?;
                 }
 
-                for batch in data_batches {
-                    if batch.num_rows() > 0 {
-                        sqlite.insert_batch(&transaction, batch, on_conflict.as_ref())?;
+                while let Some(data_batch) = batch_rx.blocking_recv() {
```

Comment:
@peasee - can we perform a read query while streaming results?

Reply:
You can perform read queries for tables except the ones still accelerating. If you try querying a table that's accelerating, the query will hang until the acceleration finishes before returning values.

This is the existing behavior on `main`.